lib/engine/game/g_1849/meta.rb

```diff
@@ -24,6 +24,27 @@ module Meta
             short_name: 'Delay IFT',
             desc: 'IFT may not be one of the first three corporations (recommended for newer players)',
           },
+          {
+            sym: :reduced_4p_corps,
+            short_name: '(Alpha) - Reduced corps at 4 players',
```

Comment:
`()` and `-` are redundant. Suggest removing the `-` in all of these.

Reply:
Done.

Reply:
commit name: `meta fix`